### PR TITLE
✨ Type definitions for cli-welcome package

### PR DESCRIPTION
**Pull request by max-programming**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by ahmadawais — APPROVED** [review]
Looks good to me! ✔︎

File: types/cli-welcome/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    tagLine: string;
+    description?: string;
+    version?: string;
+    bgColor?: string;
+    color?: string;
+    clear?: boolean;
+    bold?: boolean;
+}
+
+declare function welcome(options: WelcomeOptions): void;
```
Comment (peterblazejewicz) [inline types/cli-welcome/index.d.ts:17]:
```ts
declare function welcome(options?: WelcomeOptions): void;
```
`options `is an optional param and all props are optional, taking defaults:
https://github.com/ahmadawais/cli-welcome/blob/master/index.js#L13
I'm givin OK as  @ahmadawais revied this alrady, thx!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by max-programming** [comment]
Ready to merge

**Pull request merged**